tests/tests/fv3_ccpp_gsd_sar_v1

```diff
@@ -0,0 +1,37 @@
+###############################################################################
```

Comment:
If we are adding a new test fv3_ccpp_gsd_sar_v1, then we will also want to run it as part of rt_ccpp_gsd.conf, or?